### PR TITLE
[prometheus-cpp] Update to version 1.0.0

### PR DESCRIPTION
**Pull request by gjasny**
Update prometheus-cpp to 1.0.0. Locally tested all the features on macOS.

### CODE REVIEW

File: ports/prometheus-cpp/portfile.cmake [hunk 1]
```diff
@@ -3,38 +3,33 @@ vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
...
+    FEATURES
+        compression ENABLE_COMPRESSION
+        pull ENABLE_PULL
+        push ENABLE_PUSH
+        tests ENABLE_TESTING
+)
 
-vcpkg_configure_cmake(
+vcpkg_cmake_configure(
     SOURCE_PATH ${SOURCE_PATH}
```
Comment (JonLiu1993) [inline ports/prometheus-cpp/portfile.cmake:21]:
```suggestion
    SOURCE_PATH "${SOURCE_PATH}"
```

File: ports/prometheus-cpp/portfile.cmake [hunk 2]
```diff
@@ -3,38 +3,33 @@ vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
...
 
-vcpkg_install_cmake()
+vcpkg_cmake_install()
 
 vcpkg_copy_pdbs()
 
-vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/prometheus-cpp)
+vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/prometheus-cpp)
 
 file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
```
Comment (JonLiu1993) [inline ports/prometheus-cpp/portfile.cmake:34]:
```suggestion
file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
```

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

**Comment by JonLiu1993** [comment]
@gjasny ,Have you tested the features locally?

**Comment by gjasny** [comment]
> @gjasny ,Have you tested the features locally?

Yes, locally on my mac.

**Review by JonLiu1993 — APPROVED** [review]

**Comment by JonLiu1993** [comment]
All features are tested successfully in the following triplet:
- x86-windows
- x64-windows
- x64-windows-static

**Pull request merged**